pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3876,6 +3876,34 @@ int BPF_KPROBE(trace_do_sigaction)
     return events_perf_submit(&p, DO_SIGACTION, 0);
 }
 
+SEC("kprobe/do_truncate")
+int BPF_KPROBE(trace_do_truncate)
+{
+    program_data_t p = {};
+    if (!init_program_data(&p, ctx))
+        return 0;
+
+    if (!should_trace(&p))
+        return 0;
+
+    if (!should_submit(DO_TRUNCATE, &(p.event->context)))
+        return 0;
+
+    struct dentry *dentry = (struct dentry *) PT_REGS_PARM2(ctx);
+    long length = (long) PT_REGS_PARM3(ctx);
```

Comment:
length has type `loff_t`, which for me translates into an `unsigned long long` (u64) and not a `signed long` (s32) like here. no ?